src/test/isolation2/sql/vacuum_progress_row.sql

```diff
@@ -136,6 +136,9 @@ select relid::regclass as relname, phase, heap_blks_total, heap_blks_scanned, he
 
 -- Resume execution of compact phase and block at syncrep.
 2: SELECT gp_inject_fault_infinite('wal_sender_loop', 'suspend', dbid) FROM gp_segment_configuration WHERE role = 'p' and content = 1;
+-- Dispatcher shouldn't proceed until status_version in FtsVersion in FtsProbeInfo is updated,
+-- ensuring the backends are restarted before they start the post-cleanup phase.
+2: SELECT gp_inject_fault('vacuum_rel_finished_one_relation', 'suspend', '', '', 'vacuum_progress_ao_row', 1, 1, 0, 1) FROM master();
 2: SELECT gp_inject_fault('vacuum_ao_after_compact', 'reset', dbid) FROM gp_segment_configuration WHERE content > -1 AND role = 'p';
```

Comment:
Why not reset after we find that the mirror is down?

Reply:
That would change the original behavior of the test: the segments block on syncrep after `vacuum_ao_after_compact` but before `vacuum_rel_finished_one_relation`. If we reset `vacuum_ao_after_compact` after mirror is down, the segment will immediately skip syncrep. I think it's better to suspend on `vacuum_rel_finished_one_relation` since it's positioned roughly at the point where the segments are expected to be without the synchronization (although this was flaky)